Ports/julius/package.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='julius'
+version='1.7.0'
+useconfigure='true'
+files=(
+    "https://github.com/bvschaik/julius/archive/refs/tags/v${version}.tar.gz 3ee62699bcbf6c74fe5a9c940c62187141422a9bd98e01747a554fd77483431f"
+)
+depends=(
+    "cmake" 
```

Comment:
Sorry, I forgot about this one in my earlier review. We generally don't include build dependencies here. Currently, the user is just assumed to have all build dependencies installed on their host machine.